ios/Sources/HomeFeature/HomeFeature.swift

```diff
@@ -1,11 +1,46 @@
+import Component
 import ComposableArchitecture
 
+// dummy feed model
+public struct FeedItem: Equatable, Identifiable {
+    public var id: String
+    public var imageURLString: String
+    public var link: String
+    public var media: TagType
+    public var publishedAt: Date
+    public var summary: String
+    public var title: String
+
+    public init(id: String, imageURLString: String, link: String, media: TagType, publishedAt: Date, summary: String, title: String) {
+        self.id = id
+        self.imageURLString = imageURLString
+        self.link = link
+        self.media = media
+        self.publishedAt = publishedAt
+        self.summary = summary
+        self.title = title
+    }
+}
+
 public struct HomeState: Equatable {
     // TODO: Replace to real models
-    public var contents: [String]
+    private var feedItems: [FeedItem]
+    public var message: String
+
+    public var topic: FeedItem? {
+        feedItems.first
+    }
+
+    public var listFeedItems: [FeedItem] {
+        Array(feedItems.dropFirst())
+    }
```

Comment:
These values look like not called from more than one place. This could be removed or add as extension in HomeScreen.swift file I think.

Reply:
I don't think so 😢 

> These values look like not called from more than one place.

It's not about reusability, it's just an explicit indication of what the property is for.

- With no extensions about this:
  - There will be a little logics in View such as `ForEach(feedItems.dropFirst())`. In my opinion, View should be just for display.

- With extensions in `HomeScreen`
  - In my opinion, I think `State` properties should be placed in own `struct` or `class` because to create extension in other files, it isn't intuitive.